flink-runtime/src/test/java/org/apache/flink/runtime/operators/sort/MergeIteratorTest.java

```diff
@@ -104,15 +105,15 @@ public void testMergeOfTwoStreams() throws Exception {
 
         int pos = 1;
 
-        Assert.assertTrue((rec1 = iterator.next(rec1)) != null);
-        Assert.assertEquals(expected[0], rec1.f0.intValue());
+        assertThat(rec1 = iterator.next(rec1)).isNotNull();
+        assertThat(rec1.f0.intValue()).isEqualTo(expected[0]);
```

Comment:
You can help search similar code globally in `operators` package, and I won't comment them, thanks~

Reply:
Nice catch!
The `intValue()` is unnecessary because `rec1.f0` is Interger type.